src/components/GameAbout/index.tsx

```diff
@@ -26,6 +32,17 @@ const GameAbout = ({ description, classnames }: GameAboutProps) => {
 
   return (
     <div className={classnames?.container}>
+      {gameName && (
+        <div
```

Comment:
Maybe we should use a header tag instead of a div.

Reply:
for this one elio meant the tag should be one of `h1`, `h2`, `h3`... `h6`. The `header` tag is meant for a different use case.

From https://www.w3schools.com/tags/tag_header.asp
>A `<header>` element typically contains:
> one or more heading elements `(<h1> - <h6>)`
> logo or icon
> authorship information


If we want to use the `header` css class, then it could still be one of `h1-h6`, but by specifying the `header` css class, it would override the default styling while keeping it semantic.

In general, we aren't very semantic though so I'm fine with it being a div